From b326f9aebe7d4337456259673ab31bad86771c2b Mon Sep 17 00:00:00 2001
From: Achill Gilgenast <achill@achill.org>
Date: Tue, 2 Dec 2025 13:22:49 +0100
Subject: [PATCH] tools/nftables.service: use /etc/nftables.nft

In Alpine /etc/nftables.nft is used as the main nftrules file. To match
OpenRC behavior and existing rules, use that aswell.

In the future, we might want to revisit this and follow upstream.

Signed-off-by: Achill Gilgenast <achill@achill.org>
---
 tools/nftables.service.in | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/tools/nftables.service.in b/tools/nftables.service.in
index 2ac7e6fd0031..7b967a0d1108 100644
--- a/tools/nftables.service.in
+++ b/tools/nftables.service.in
@@ -5,7 +5,7 @@ Wants=network-pre.target
 Before=network-pre.target shutdown.target
 Conflicts=shutdown.target
 DefaultDependencies=no
-ConditionPathExists=@pkgsysconfdir@/rules/main.nft
+ConditionPathExists=/etc/nftables.nft
 
 [Service]
 Type=oneshot
@@ -13,8 +13,8 @@ RemainAfterExit=yes
 StandardInput=null
 ProtectSystem=full
 ProtectHome=true
-ExecStart=@sbindir@/nft 'flush ruleset; include "@pkgsysconfdir@/rules/main.nft"'
-ExecReload=@sbindir@/nft 'flush ruleset; include "@pkgsysconfdir@/rules/main.nft"'
+ExecStart=@sbindir@/nft 'flush ruleset; include "/etc/nftables.nft"'
+ExecReload=@sbindir@/nft 'flush ruleset; include "/etc/nftables.nft"'
 ExecStop=@sbindir@/nft flush ruleset
 
 [Install]
-- 
2.52.0

